examples/ocPredprobDist.R

```diff
@@ -41,10 +41,14 @@ result$oc
 # Desired difference to Standard of Care for Efficacy and Futility is 10% and -10% respectively.
 # Grey zone occurs due to different posterior probability distribution in the Efficacy and Futility rules.
 # Delta calculation is absolute case. The following are the Final Stop rules respectively :
-# - Final look for Efficacy: Pr( response rate + deltaE > 25% ) > 60% or P(response rate + deltaE > p0) > tT
-# - Final look for Futility: Pr( response rate + deltaF < 25% ) < 60% or P(response rate + deltaF > p0) < tT
-# - Interim look for Efficacy: Pr( success at final ) > 80% or P(success at final) > phiU
-# - Interim look for Futility: Pr( failure at final ) < 20% or P(success at final) < phiL
+# - Final look for Efficacy: Pr( response rate + deltaE > 25% ) > 60% or
```

Comment:
better use `RR` instead of `response rate` in the formulas in general (then it is easier to read as well)